andela/ah-leagueOfLegends
# #159965477 Send email for verification

## What does this PR do?
```
Adds email verification by sending email to the user
```

## Description of Task to be completed?
```
Once the user signs up, an email address is sent to the email address the user 
signed up with

```

## How should this be manually tested?

```
1. Clone this repository
2. Checkout ft-send-email-verification-159965477
3. pip install -r requirements.txt this should also install celery
4. If on a Mac, run `brew install rabbitmq` to install rabbitmq message brocker
5. Run rabbitmq using `brew services start rabbitmq`
6. Run `celery -A authors worker -l info` to start celery
7. Run the server python manage.py runserver
8. navigate to 127.0.0.1:8000/api/users
9. Signup using your email address
10. Check your inbox, there should be a verification email

```
## Any background context you want to provide?

```
When users signup to our application, we want to ensure that the email used is valid. 
Therefore send a token to the email used to register and the user has to click on 
the link in the email to activate their account

```

## What are the relevant pivotal tracker stories?

[#159965477](https://www.pivotaltracker.com/story/show/159965477)


## Screenshots if available to support PR.

<img width="949" alt="screen shot 2018-09-07 at 19 58 47" src="https://user-images.githubusercontent.com/7246507/45232838-8c856580-b2d9-11e8-9124-518be918cfb9.png">
